pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -39,210 +40,29 @@ var gatheringFunctionBuilders = map[GatheringFunctionName]GathererFunctionBuilde
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherAPIRequestCounts: GatherAPIRequestCountsParams{
-				AlertName: "APIRemovedInNextEUSReleaseInUse",
-			},
-		},
-	},
-	// GatherContainersLogs
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "KubePodCrashLooping",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "KubePodCrashLooping",
-				TailLines: 20,
-				Previous:  true,
-			},
-		},
-	},
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "KubePodNotReady",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "KubePodNotReady",
-				TailLines: 100,
-				Previous:  false,
-			},
-			GatherPodDefinition: GatherPodDefinitionParams{
-				AlertName: "KubePodNotReady",
-			},
-		},
-	},
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "AlertmanagerFailedToSendAlerts",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "AlertmanagerFailedToSendAlerts",
-				Container: "alertmanager",
-				TailLines: 50,
-			},
-		},
-	},
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "PrometheusOperatorSyncFailed",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "PrometheusOperatorSyncFailed",
-				Container: "prometheus-operator",
-				TailLines: 50,
-			},
-		},
-	},
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "AlertmanagerFailedReload",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "AlertmanagerFailedReload",
-				Container: "alertmanager",
-				TailLines: 50,
-			},
-		},
-	},
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "PrometheusTargetSyncFailure",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "PrometheusTargetSyncFailure",
-				Container: "prometheus",
-				TailLines: 50,
-			},
-		},
-	},
-	{
-		Conditions: []ConditionWithParams{
-			{
-				Type: AlertIsFiring,
-				Alert: &AlertConditionParams{
-					Name: "ThanosRuleQueueIsDroppingAlerts",
-				},
-			},
-		},
-		GatheringFunctions: GatheringFunctions{
-			GatherContainersLogs: GatherContainersLogsParams{
-				AlertName: "ThanosRuleQueueIsDroppingAlerts",
-				Container: "thanos-ruler",
-				TailLines: 50,
-			},
-		},
-	},
-}
-
```

Comment:
Some of these conditions are not currently defined by the new service. Can you please create a task to update it?

Reply:
https://github.com/RedHatInsights/insights-operator-gathering-conditions/pull/10
I'll also update the tag in the service once this one is merged to trigger a new build of the service

Reply:
Thank you

Reply:
The service update PR https://github.com/RedHatInsights/insights-operator-gathering-conditions-service/pull/140